fio.spec

```diff
@@ -86,6 +86,9 @@ rm -rf $RPM_BUILD_ROOT
 %{_usrsrc}/%{name}-%{version}
 
 %changelog
+* Thu Nov 11 2021 Wang Shilong <shilong.wang@intel.com> 3.20-6
+- Rebuilt for breaking DAOS API change
```

Comment:
Should add that you updated to a newer release here too.

Reply:
Yup, sure, btw, there is no Centos8 RPMS for results..